### ISSUE TITLE
Autokeras does not save model

### ISSUE THREAD

**Issue by uldcap** [keras-team/autokeras#1129]
Hello everyone,

I am trying to save an autokeras StructuredDataRegressor model.
I am doing some test since my training is a very long one. Below you have the code and the error. Please, can you help me?

**CODE**
```
import autokeras as ak
import tensorflow as tf

regressor=ak.StructuredDataRegressor(max_trials=2)
regressor.fit(x=x_train, y=y_train, epochs=2)

model=regressor.export_model()
try:
	model.save('model_autokeras', save_format='tf')
except:
	model.save('model_autokeras.h5')
```

**ERROR**
```
Traceback (most recent call last):
  File "Autokeras_test.py", line 54, in <module>
    model.save('model_autokeras', save_format='tf')
  File "/data//miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/network.py", line 1008, in save
    signatures, options)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/save.py", line 115, in save_model
    signatures, options)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/saved_model/save.py", line 78, in save
    save_lib.save(model, filepath, signatures, options)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/saved_model/save.py", line 909, in save
    meta_graph_def, saveable_view, signatures, options.namespace_whitelist)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/saved_model/save.py", line 553, in _fill_meta_graph_def
    object_map, resource_map, asset_info = saveable_view.map_resources()
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/saved_model/save.py", line 251, in map_resources
    new_resource = obj._create_resource()
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/ops/lookup_ops.py", line 1932, in _create_resource
    name=self._name)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/ops/gen_lookup_ops.py", line 1113, in mutable_dense_hash_table_v2
    max_load_factor=max_load_factor, name=name)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/framework/op_def_library.py", line 468, in _apply_op_helper
    preferred_dtype=default_dtype)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/framework/ops.py", line 1280, in convert_to_tensor
    raise RuntimeError("Attempting to capture an EagerTensor without "
RuntimeError: Attempting to capture an EagerTensor without building a function.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "Autokeras_test.py", line 56, in <module>
    model.save('model_autokeras.h5')
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/engine/network.py", line 1008, in save
    signatures, options)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/save.py", line 112, in save_model
    model, filepath, overwrite, include_optimizer)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/hdf5_format.py", line 109, in save_model_to_hdf5
    save_weights_to_hdf5_group(model_weights_group, model_layers)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/saving/hdf5_format.py", line 627, in save_weights_to_hdf5_group
    weight_values = K.batch_get_value(weights)
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/backend.py", line 3270, in batch_get_value
    return [x.numpy() for x in tensors]
  File "/data/miniconda3/envs/w_autokeras/lib/python3.7/site-packages/tensorflow_core/python/keras/backend.py", line 3270, in <listcomp>
    return [x.numpy() for x in tensors]
AttributeError: 'TrackableWeightHandler' object has no attribute 'numpy'
```

**Comment by utkarshgupta137** [comment]
Facing the same issue.

**Comment by haifeng-jin** [comment]
This is fixed in 1.0.3. Please let me know if it is not.

**Comment by utkarshgupta137** [comment]
Issue solved in the latest version, thanks.
